Atomic/util.py

```diff
@@ -341,8 +342,12 @@ def skopeo_manifest_digest(manifest_file, debug=False):
     cmd = cmd  + ['manifest-digest', manifest_file]
     return check_output(cmd).rstrip().decode()
 
-def skopeo_copy(source, destination, debug=False, sign_by=None, insecure=False):
-    cmd = [SKOPEO_PATH]
+def skopeo_copy(source, destination, debug=False, sign_by=None, insecure=False, policy_filename=None):
+
+    if not policy_filename:
+        policy_filename=POLICY_FILENAME
+
+    cmd = [SKOPEO_PATH, "--policy=%s" % policy_filename]
```

Comment:
I’d be a bit happier if `atomic` were only overriding `--policy` if it had a _non-default_ value; this would minimize the risk of `skopeo` and `atomic` accidentally using a different path (see the `policy.conf` typo which happened earlier in this PR.)